examples/cc26xx/cc26xx-web-demo/cc26xx-web-demo.c

```diff
@@ -966,6 +996,57 @@ PROCESS_THREAD(cc26xx_web_demo_process, ev, data)
 
   PROCESS_END();
 }
+
+PROCESS_THREAD(adc_process, ev, data)
+{
+  PROCESS_BEGIN();
+  static struct etimer et_adc;
+  while(1)
+  {
+	etimer_set(&et_adc, CLOCK_SECOND*5);
+	PROCESS_WAIT_EVENT(); 
+	if(etimer_expired(&et_adc)) {
+		//intialisation of ADC
+		ti_lib_aon_wuc_aux_wakeup_event(AONWUC_AUX_WAKEUP);
+		while(!(ti_lib_aon_wuc_power_status_get() & AONWUC_AUX_POWER_ON))
+		{ }
+
+		// Enable clock for ADC digital and analog interface (not currently enabled in driver)
+		// Enable clocks
+		ti_lib_aux_wuc_clock_enable(AUX_WUC_ADI_CLOCK | AUX_WUC_ANAIF_CLOCK | AUX_WUC_SMPH_CLOCK);
+		while(ti_lib_aux_wuc_clock_status(AUX_WUC_ADI_CLOCK | AUX_WUC_ANAIF_CLOCK | AUX_WUC_SMPH_CLOCK) != AUX_WUC_CLOCK_READY)
+		{ }
+		//printf("clock selected\r\n");
+
+		// Connect AUX IO7 (DIO23, but also DP2 on XDS110) as analog input.
+		AUXADCSelectInput(ADC_COMPB_IN_AUXIO7); 
+		//printf("input selected\r\n");
+
+		// Set up ADC range
+		// AUXADC_REF_FIXED = nominally 4.3 V
+		AUXADCEnableSync(AUXADC_REF_FIXED,  AUXADC_SAMPLE_TIME_2P7_US, AUXADC_TRIGGER_MANUAL);
+		//printf("init adc --- OK\r\n");
+
+		//Trigger ADC converting
+		AUXADCGenManualTrigger();
+		//printf("trigger --- OK\r\n");
+
+		//reading adc value
+		single_adc_sample = AUXADCReadFifo();
+                single_adc_sample = (4300*single_adc_sample)/4096;
```

Comment:
if single_adc_sample should ever get bigger than ~16 we're overflowing the 16 bit variable!